docs/README.md

```diff
@@ -187,6 +190,57 @@ is installed. `Succeeded` should appear in the **Status** column.
 
 If you're not using OpenShift or simply want to install from source, follow the [instructions in the Contributor Documentation](../README.md#deploying-the-operator).
 
+## Migrating from Istio in-cluster Operator
```

Comment:
Should we be adding also an example of the entire process? Or is too much information for this kind of docs?

Reply:
I think we don't want to offer a complete migration path at this stage... You can of course do a revision-based update from your existing control plane to the one deployed by Sail Operator, but I'd say we let people figure it out. If we can add a more detailed guide down the line that'll be great, but the important part for now I think is that we point out how to recreate the same instance, people can take it from there